_search-plugins/concurrent-segment-search.md

```diff
@@ -34,11 +34,14 @@ The index-level setting takes priority over the cluster-level setting. Thus, if
 
 Both the cluster- and index-level `search.concurrent_segment_search.mode` settings accept the following values:
 
-- `all`: Enables concurrent segment search across all search requests. This is equivalent to setting `search.concurrent_segment_search.enabled` to `true`.
+- `all`: Enables concurrent segment search across all search requests. This is equivalent to setting `search.concurrent_segment_search.enabled` to `true`. Concurrent segment search is disabled by default in this mode.
```

Comment:
We don't need `Concurrent segment search is **disabled** by default in this mode`. It is also incorrect since concurrent search is enabled in this mode for all requests.

Reply:
@sohami To clarify, this original sentence `Starting with OpenSearch version 3.0, concurrent search will be enabled by default in "auto" mode on the cluster` doesn't mean that it's enabled _only_  in the `auto` mode (and disabled for all other modes). It means that it's enabled for all modes, and the default mode is `auto`?